docs/reference/settings/monitoring-settings.asciidoc

```diff
@@ -198,11 +198,16 @@ xpack.monitoring.exporters:
 
 `auth.username`::
 
-The username is required if a `auth.password` is supplied.
+The username is required if `auth.secure_password` or `auth.password` is supplied.
+
+`auth.secure_password` ({ref}/secure-settings.html[Secure], {ref}/secure-settings.html#reloadable-secure-settings[reloadable])::
+
+The password for the `auth.username`. Takes precedence over `auth.password` if it is also specified.
 
 `auth.password`::
 
-The password for the `auth.username`.
+The password for the `auth.username`. This setting is ignored if `auth.secure_password` is also specified. This setting is
```

Comment:
In general, we put the conditional part of the sentence at the beginning so folks can stop reading if it doesn't apply to them.
```suggestion
The password for the `auth.username`. If `auth.secure_password` is also specified, this setting is ignored.
```